lesson03/assignment/index.html

```diff
@@ -3,13 +3,35 @@
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<link rel="stylesheet" href="css/style.css" />
 	<title>Fun with Constructors!</title>
+	<script src= "https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
 	<script src="index.js"></script>
 </head>
 <body>
-	<h1>Assignment 3</h1>
-	<section class="template-hook">
-		Fill this section with the templated components
+	
```

Comment:
For this assignment, we were looking to have the whole component come from your JavaScript template. So for here, you'll want just some sort of container like a div with a class that you can populate from your JavaScript.

Reply:
oh yes - this was the key concept I was missing - so stupid on me - I was trying to figure out how I was going to get the think to loop.   But changed a couple things and boom worked. ;-)